roles/utils_build_push_image/vars/main/resources.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Script for building and pushing images
+utils_create_build_config_script: roles/utils_build_push_image/files/create-build-config.sh
+build_config_output_dir: roles/utils_build_push_image/files
```

Comment:
output dir should always be `{{ artifact_extra_logs_dir }}`, except when dealing with secret information (you're not, at this point)